src/EnergyPlus/DaylightingManager.hh

```diff
@@ -543,6 +543,74 @@ struct DaylightingManagerData : BaseGlobalStruct {
     Array2D_int MapErrIndex;
     Array2D_int RefErrIndex;
 
+    bool MySunIsUpFlag = false;
+    bool CalcDayltgCoeffsMapPointsMySunIsUpFlag = false;
+
+    // static variables extracted from functions
+    Vector3<Real64> AR;  // Inside surface area sum for floor/wall/ceiling (m2)
```

Comment:
I think a much better design would be to have an actual DaylightingManager object that holds data and persists it, much like the input processor, SQLite manager, or plugin manager.  But this is fine for now to at least get these statics off of the global state.